src/map.cpp

```diff
@@ -175,24 +174,24 @@ void Map::removeTile(uint16_t x, uint16_t y, uint8_t z)
 			}
 		}
 
-		Item* ground = tile->getGround();
+		auto ground = tile->getGround();
```

Comment:
Could this be `const auto`
```suggestion
		const auto ground = tile->getGround();
```